### ISSUE TITLE
missing dependency?

### ISSUE THREAD

**Issue by jedahan** [zigalenarcic/mangl#3]
I installed all the deps mentioned on ubuntu 18.04, but make cannot find glut.pc. I searched `/` but that file doesn't exist. Is there another package I should be installing?

```
 ▲ ~/src/mangl make                                                                                            master 1d ⬡
Package glut was not found in the pkg-config search path.
Perhaps you should add the directory containing `glut.pc'
to the PKG_CONFIG_PATH environment variable
No package 'glut' found
Package glut was not found in the pkg-config search path.
Perhaps you should add the directory containing `glut.pc'
to the PKG_CONFIG_PATH environment variable
No package 'glut' found
cc -g -O2 -Wall  -o mangl mandoc/compat_err.o mandoc/compat_fts.o mandoc/compat_getline.o mandoc/compat_getsubopt.o mandoc/compat_isblank.o mandoc/compat_mkdtemp.o mandoc/compat_ohash.o mandoc/compat_progname.o mandoc/compat_reallocarray.o mandoc/compat_recallocarray.o mandoc/compat_strcasestr.o mandoc/compat_strlcat.o mandoc/compat_strlcpy.o mandoc/compat_strndup.o mandoc/compat_strsep.o mandoc/compat_strtonum.o mandoc/compat_vasprintf.o mandoc/man.o mandoc/man_macro.o mandoc/man_validate.o mandoc/att.o mandoc/lib.o mandoc/mdoc.o mandoc/mdoc_argv.o mandoc/mdoc_macro.o mandoc/mdoc_state.o mandoc/mdoc_validate.o mandoc/st.o mandoc/eqn.o mandoc/roff.o mandoc/roff_validate.o mandoc/tbl.o mandoc/tbl_data.o mandoc/tbl_layout.o mandoc/tbl_opts.o mandoc/arch.o mandoc/chars.o mandoc/mandoc.o mandoc/mandoc_aux.o mandoc/mandoc_msg.o mandoc/mandoc_ohash.o mandoc/mandoc_xr.o mandoc/msec.o mandoc/preconv.o mandoc/read.o mandoc/tree.c mandoc/mdoc_term.c mandoc/man_term.c mandoc/tbl_term.c mandoc/tag.c mandoc/roff_term.c mandoc/eqn_term.c mandoc/term_ascii.c mandoc/term.c mandoc/term_tab.c mandoc/out.c hashmap.c main.c -lm
main.c:30:10: fatal error: ft2build.h: No such file or directory
 #include "ft2build.h"
          ^~~~~~~~~~~~
compilation terminated.
Makefile:72: recipe for target 'mangl' failed
make: *** [mangl] Error 1
```

**Comment by jedahan** [comment]
related, 9 year old debian bug? https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=575025

**Comment by jedahan** [comment]
I had to make these two changes to build, they seem to be two separate issues.

```
diff --git a/Makefile b/Makefile
index b9f2c0d..41137ad 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-CFLAGS = -g -O2 -Wall $(shell pkg-config --cflags zlib gl glut freetype2)
-LDFLAGS = -lm $(shell pkg-config --libs zlib gl glut freetype2)
+CFLAGS = -g -O2 -Wall $(shell pkg-config --cflags zlib gl freetype2)
+LDFLAGS = -lm -lglut $(shell pkg-config --libs zlib gl freetype2)
 
 LIBMAN_OBJS	 = mandoc/man.o \
 			   mandoc/man_macro.o \
diff --git a/main.c b/main.c
index 33738a4..2e6e716 100644
--- a/main.c
+++ b/main.c
@@ -135,7 +135,7 @@ FontData builtinFont = {
     .bitmap_height = 0,
     .character_width = 6, // actual width and height
     .character_height = 9,
-    .line_height = font_char_height,
+    .line_height = 14,
     .font_size = 10.0,
     .texture_id = 0
 };
```

**Comment by zigalenarcic** [comment]
Ok thanks,
I'll change the Makefile. I'm new to pkg-config, it did work on Alpine Linux and I didn't check it on Ubuntu.
The second issue seems to be connected to (gcc or clang) compiler version. I'll try to fix that too.

**Comment by zigalenarcic** [comment]
Fixed in 845acd7

**Comment by jedahan** [comment]
Awesome, thanks!
